dynamic_instrumentation/src/main.rs

```diff
@@ -48,6 +48,15 @@ struct Args {
     #[clap(long, value_parser)]
     metadata: PathBuf,
 
+    /// Path to the `c2rust-analysis-rt` crate if you want to use a local version of it (vs. the crates.io one).
+    /// This is not used unless `--set-runtime` is also passed.
+    #[clap(long, value_parser)]
+    runtime: Option<PathBuf>,
```

Comment:
I would call this `runtime_path` or something.

Reply:
I had named it `--runtime` since it was previously named that, before the `cargo`/`$RUSTC_WRAPPER` refactor.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/562/commits/9d89818f44016348ff96a554c646f6ba53a03198.